tests/BasicRum/Beacon/UserAgentImportTest.php

```diff
@@ -43,10 +45,25 @@ public function testUserAgentSimpleImport()
                     ],
-                ]
-            ));
+                ];
+
+                foreach ($array as $key => $item)
```

Comment:
The previous tests were failing because we forgot to remove this code. It tries to simulate that 'restiming' was decoded in catcher.php but we discussed that we should not do this. restiming will be decoded on beacon import.